### ISSUE TITLE
Laravel Post Update Hooks Bug

### ISSUE THREAD

**Issue by mallardduck** [mallardduck/laravel-HumanoID#3]
> Figure out WTF caused this...

```
> @php artisan ide-helper:generate
Exception: The application is missing the wordsets folder, publish the `humanoid` resources first.
Skipping \MallardDuck\LaravelHumanoID\Facades\HumanoID.
Exception: The application is missing the wordsets folder, publish the `humanoid` resources first.
Skipping \MallardDuck\LaravelHumanoID\Facades\HumanoID.
```

**Comment by emmadesilva** [comment]
I assume the wordsets folder is shipped in the vendor directory right? Could it be that the ide-helper is searching for the wordsets in the root Laravel directory?

**Comment by mallardduck** [comment]
I think it's some sort of race condition TBH. Here's where it's throwing from:
https://github.com/mallardduck/laravel-HumanoID/blob/main/src/HumanoIDManager.php#L86

Basically my logic here was that "applications should own their words file" - so by default (when someone publishes the vendor files) I copy the two wordsets shipped by HumanoID into `resources/humanoid`.

I believe what `ide-helper:generate` is doing here is learning about the Facades in my app so as to generate a single `_ide_helper.php` file. In theory we can ignore this as mark it as a non-issue, however it could be nice to fix if possible. The main catch would be I'd prefer to fix it without exposing new sharp edges that this currently prevents.

**Comment by mallardduck** [comment]
Actually - think I've decided I'm content with this being a `Wont Fix`.

This only affects the ide-generator before you configure the package. So while the `_ide_helper.php` file won't include everything from this package, it's not broken. This really only affects the IDE hints for the alias of the Facade too. Plus once a user properly configures the package once, then any time after that the `ide-helper:generate` will work properly.

Time is probably better spent adding the "setup package" console command I imagined. This console command will help users configure the package and will copy/modify the `DefaultAppConfig` into their project.
